script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -309,6 +310,7 @@ The small, iron box hangs agape.
 [sha:7dba6b00c979de7024176adaaa0bb3ba3aead616]{
 -- Page 737, Offset 7561.
 -- 箱のなかには、
+// i know it's intended damn that looks like a linebreak error
```

Comment:
I toyed with progressive indent, could do that to make it more clearly intentional

Reply:
maybe affix ellipses to the end of each, or 3x cjk. I agree that it looks pretty bad but english doesn't have particles like the jp does so we need to do _something_ there.